cmd/noms/noms.go

```diff
@@ -203,13 +205,22 @@ See Spelling Objects at https://github.com/attic-labs/noms/blob/master/doc/spell
 	show.Flag("stats", "If true, reports statistics related to the value").Bool()
 	show.Arg("object", "a noms object").Required().String()
 
+	// sync
 	sync := noms.Command("sync", `Moves datasets between or within databases
 See Spelling Objects at https://github.com/attic-labs/noms/blob/master/doc/spelling.md for details on the object and dataset arguments.
 `)
 	sync.Flag("parallelism", "").Short('p').Default("512").Int()
 	sync.Arg("source-object", "a noms source object").Required().String()
 	sync.Arg("dest-dataset", "a noms dataset").Required().String()
 
+	// splore
+	splore := noms.Command("splore", `Interactively explore a Noms database using a web browser.
+See Spelling Objects at https://github.com/attic-labs/noms/blob/master/doc/spelling.md for details on the path argument.
+`)
+	splore.Flag("port", "Server port. Defaults to a random port.").Short('p').Int()
```

Comment:
why not lsof the port and increment, or require a port arg if not 8000 or whatever default is? Seems like it would be annoying to script this if the port is not dependable, maybe it's a moot point because of the `--browser` flag though